erpnext/accounts/doctype/account/account.py

```diff
@@ -89,7 +89,7 @@ def validate_root_details(self):
 				throw(_("Root cannot be edited."), RootNotEditable)
 
 		if not self.parent_account and not self.is_group:
-			frappe.throw(_("Root Account must be a group"))
+			frappe.throw(_("Root Account must be a group for {0}".format(frappe.bold(self.name))))
```

Comment:
```suggestion
			frappe.throw(_("Root Account must be a group for {0}").format(frappe.bold(self.name)))
```
This wont break translations. Refer [this](https://frappe.io/docs/user/en/translations) for more information.